velox/docs/functions/spark/array.rst

```diff
@@ -15,6 +15,15 @@ Array Functions
 
         SELECT array(1, 2, 3); -- [1,2,3]
 
+.. spark:function:: array_append(array(E), element) -> array(E)
+
+    Add the ``element`` at the end of the ``array`` passed as first argument.
+    Type of element should be same to type of the elements of the ``array``.
+    NULL element is also appended into the array. But if the ``array`` passed is NULL output is NULL. ::
```

Comment:
> But if the ``array`` passed is NULL output is NULL.

Comment suggestion: Returns NULL when the input \``array\`` is NULL.

Reply:
Done